src/khoj/routers/email.py

```diff
@@ -47,3 +46,38 @@ async def send_welcome_email(name, email):
             "html": html_content,
         }
     )
+
+
+from pydantic import BaseModel
+
+
+class TextData(BaseModel):
+    uquery: str
+    kquery: str
+    sentiment: str
+
+
+async def feedback(uquery, kquery, sentiment):
+    # console debug messages
+    logger.debug("SENDING USER FEEDBACK...")
+    logger.debug(f"User Query: {uquery}\n")
+    logger.debug(f"Khoj Response: {kquery}\n")
+    logger.debug(f"Sentiment: {sentiment}\n")
+    # rendering feedback email using feedback.html as template
+    template = env.get_template("feedback.html")
+    html_content = template.render(
+        uquery=uquery if not is_none_or_empty(uquery) else "N/A",
+        kquery=kquery if not is_none_or_empty(kquery) else "N/A",
+        sentiment=sentiment if not is_none_or_empty(sentiment) else "N/A",
+    )
+    # send feedback from two fixed accounts
+    r = resend.Emails.send(
```

Comment:
Can you add a check here to see if resend is enabled before making the template/sending the email? If it's not enabled, then add the feedback to a log file, like `query_feedback.log`. Since some of our users are self-hosted, they might still want to aggregate the feedback, but won't be able to directly send the email.

Reply:
Update, let's address the local feedback log file later on. Don't worry about it for now.